### DISCUSSION TITLE
OAuth2

### DISCUSSION THREAD

**Discussion by Lion-King287** [pelican-dev/panel#173] [Ideas]
I think it would be a relief for some people if you could optionally manage user accounts via OAuth2 (e.g. authentik NOT only Discord OAuth2), that was something I really missed with Pterodactyl.

**Comment by tieb62** [comment]
Yeah, SSO is one of the biggest missing features of Pterodactyl and it would be a HUGE advantage to support OpenID Connect (OAuth)

**Comment by lancepioch** [comment]
Oauth support is planned, thanks!
